Marlin/nozzle.h

```diff
@@ -31,7 +31,7 @@
  *
  * @todo: Do not ignore the end.z value and allow XYZ movements
  * @todo: Currently this feature needs HAS_BED_PROBE to be active
- *  due to the do_blocking_move_to*() functions.
+ *        due to the do_blocking_move_to*() functions.
```

Comment:
To ensure the function is defined, either move it out of the `#if` block or add `ENABLED(NOZZLE_PARK_FEATURE)` to the list of conditions that cause the function to become enabled.